apps-rendering/src/embed.ts

```diff
@@ -68,8 +70,12 @@ interface TikTok extends GenericFields {
 	kind: EmbedKind.TikTok;
 }
 
-interface EmailSignup extends GenericFields {
+interface EmailSignup {
 	kind: EmbedKind.EmailSignup;
+	alt: Option<string>;
+	caption: Option<string>;
```

Comment:
@JamieB-gu I added `caption` here as well as `alt`. I've used `alt` to provide the required `title` attribute on the `iframe`